src/lib/constants.js

```diff
@@ -3,6 +3,7 @@ export const CLASS_ANNOTATIONS_CREATE_DRAWING = 'bp-annotations-create--drawing'
 export const CLASS_ANNOTATIONS_CREATE_HIGHLIGHT = 'bp-annotations-create--highlight';
 export const CLASS_ANNOTATIONS_CREATE_REGION = 'bp-annotations-create--region';
 export const CLASS_ANNOTATIONS_DISCOVERABLE = 'bp-annotations-discoverable';
+export const CLASS_ANNOTATIONS_ONLY_CONTROLS_ENABLED = 'bp-annotations-only-controls--enabled';
```

Comment:
There seems to be a little bit of a mismatch between the classname style (SUIT modifier) and how it's named. `enabled` seems to indicate the state of the system, that this feature is on, but it gets toggled on and off as and when the mode changes. 

Maybe we want either `bp-annotations-only-controls` to be set once and we adjust the styles as an when other recognized annotation modes classes are added or we toggle a `.bp-annotations-only-controls.is-shown` state class to hide and show. Thoughts?

Reply:
I like this idea better.